src/Components/SingleBookView/SingleBookView.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const SingleBookView = ({ trendingBooks, coverImages, descriptions, singleBookId }) => {
+
+    const foundBook = trendingBooks.find(book => book.key === `/works/${singleBookId}`)
+
+    const foundDesc = descriptions.find(desc => desc.key === `/works/${singleBookId}`) 
```

Comment:
Same thing here - it's better for the parent to do this logic so that this component only has to know about the one description it cares about :)